src/core/featuremodel.cpp

```diff
@@ -439,6 +450,87 @@ void FeatureModel::applyVertexModelToGeometry()
   mFeature.setGeometry( mVertexModel->geometry() );
 }
 
+//! a filter just to gather all matches at the same place
+class MatchCollectingFilter : public QgsPointLocator::MatchFilter
```

Comment:
For the record, that's mostly a copy/paste from QGIS' qgsvertextool.cpp, minus the geometry cache stuff.

Reply:
maybe add a note as a comment?